dist/js/copy.js

```diff
@@ -1,8 +1,19 @@
-function copyInfo( elementId ) {
-  var element = document.getElementById( elementId );
+window._wc_siftsci_js_sender = null;
+function copyInfo( source, elementId ) {
 
-  element.select();
-  document.execCommand( 'copy' );
-  
-  element.setSelectionRange( 0, 0 ); // Clear selection.
+	var sender = window._wc_siftsci_js_sender;
+
+	if( null != sender || void 0 != sender ) {
```

Comment:
I don't know why !== does not work

Reply:
null !== sender doesn't work?

If the problem is only with `void 0 != sender`, I say delete it. It is not a necessary check.

Reply:
Wait next commit